internal/commands/user/user_state.go

```diff
@@ -0,0 +1,184 @@
+package user
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+
+	"github.com/spf13/pflag"
+)
+
+// NewCommandDisable returns a *CommandUserState which is the `user disable` command
+func NewCommandDisable() *CommandUserState {
+	return &CommandUserState{}
+}
+
+// NewCommandEnable returns a *CommandUserState which is the `user enable` command
+func NewCommandEnable() *CommandUserState {
+	return &CommandUserState{userEnable: true}
+}
+
+// CommandUserState is the `user enable/disable` command
+type CommandUserState struct {
+	userEnable  bool
+	inputs      userStateInputs
```

Comment:
[question] are we collecting flags other than `--user` here?

Reply:
AuthProviderTypes for filtering when interactive selecting users.

Reply:
Do we need the interactive selection?  I'm not sure that spec'd out...for `enable`/`disable`/`revoke`, let's just keep the inputs to `--user` for now